docs/en/explanation/implicit-indexes.rst

```diff
@@ -0,0 +1,40 @@
+Implicit indexes
+================
+
+So-called implicit indexes are named as such because they are created
+without the user asking for it. Some RDBMS will do so when you create a
+foreign key: they will create an index on the referencing table, using
+the referencing columns.
+
+Here are some RDBMS that are known to create indexes when creating a
```

Comment:
RDBMS is singular. Would it make sense to put it as “database platforms that are”?